examples/language/openmoe/test_ci.sh

```diff
@@ -1,5 +0,0 @@
-set -xe
-pip install -r requirements.txt
-
-python infer.py --model "test"
-torchrun --standalone --nproc_per_node 2 train.py --model_name "test" --batch_size 1 --num_epoch 20
```

Comment:
When can you run ci?

Reply:
ci fails due to some kernel initialization. will be updated soon